cmd/build/v2/build.go

```diff
@@ -132,6 +149,11 @@ func (bc *OktetoBuilder) Build(ctx context.Context, options *types.BuildOptions)
 			if options.EnableStages {
```

Comment:
should we move this spinner update once we will build the image? just before `bc. buildService`